CONTRIBUTING.md

```diff
@@ -12,9 +12,9 @@ Setting up the development environment is simple and easy.
 1. Install the [Node Package Manager](https://www.npmjs.com/) dependencies
 1. Install the [Bower](http://bower.io/) dependencies
 1. Execute a [Grunt](http://gruntjs.com/) build -- which the default task
-1. Run the `live` Grunt task to serve the project resources locally and automatically update when file changes are made
+1. Run the Grunt `live` task to serve the project resources locally, and automatically update when files are changed
 
-```
+````
```

Comment:
isnt this normally 3 for code block? also i think 'bash' is a valid alias for shell script highlighting